packages/frontity-chakra-theme/package.json

```diff
@@ -20,14 +20,5 @@
   "homepage": "https://github.com/chakra-ui/frontity-chakra-ui-theme#readme",
   "publishConfig": {
     "access": "public"
-  },
-  "dependencies": {
-    "@chakra-ui/core": "0.5.0",
-    "@frontity/components": "^1.1.14",
-    "@frontity/hooks": "1.1.13",
-    "@frontity/html2react": "^1.1.14",
-    "emotion-theming": "10.0.19",
-    "frontity": "^1.4.2",
-    "react-icons": "3.8.0"
```

Comment:
@Bowriverstudio, why did you move all the dependencies of the theme to the root `package.json`?

Reply:
@luisherranz, I'll move them back as I suspect it is better technically.

A few months ago, I included the package `frontity-contact-form-7` and at the time it was using a different version of frontity than myself.  This caused a conflict, which was resolved when I updated the  `frontity-contact-form-7` package.  I also noticed that if I moved the dependencies into the root I did not have maintain them.

I've created several common packages for my frontity projects (just not told mentioned them yet) and some of the more stable ones are: https://github.com/Bowriverstudio/frontity-microsoft-clarity, https://github.com/Bowriverstudio/frontity-menu, https://github.com/Bowriverstudio/frontity-gutenberg-styles

I just found it tedious to update all the packages individually and easier to have them in the root.   I suspect there is probably a better solution perhaps with lerna but I did not explore that too much.

Reply:
Ok, thanks a lot for the explanation. This is something that we have to look into. Maybe we need a `npx frontity update` tool that makes it automatic.